prow_config.yaml

```diff
@@ -62,7 +62,10 @@ workflows:
     name: dashboard-release
     job_types:
       # TODO(jlewi): Reenable on presubmit once its passing.
-      #- presubmit
+      - presubmit
+      # TODO(jlewi): We should probably create a separate postsubmit job that pushes
```

Comment:
This can be removed. I recently added the postsubmit job below that pushes the image to kubeflow-images-public.